RELEASE.md

```diff
@@ -46,12 +46,6 @@ git push origin "${TAG_NAME}"
 # Build and release
 goreleaser --clean
```

Comment:
I think this line should no longer be a part of the instructions, correct? You could instead refer to the workflow that automatically runs.

Reply:
Good catch. I removed the steps. 

The current release documentation reads well if I simply remove this step so I don't think any additional changes are needed.